run-suite-task/index.ts

```diff
@@ -19,7 +19,7 @@ async function main() {
 
   } catch (error) {
     console.error(`Task failed with an error: ${error}`)
-    task.setResult(task.TaskResult.Failed, error)
+    task.setResult(task.TaskResult.Failed, `${error}`)
```

Comment:
Was there a specific reason for this change?

Reply:
Typescript started complaining that `error` needed to be a string... There's probably a couple of ways to do this. If you'd prefer a different one, I'd totally open to it.